python/lsst/ts/linearStage/ls.py

```diff
@@ -241,21 +263,61 @@ def get_position(self):
         -------
 
         """
+        status_dictionary = None
         try:
             reply = self.send("get pos")
             logger.debug(reply)
-            if self.check_reply(reply):
-                logger.info("Position captured - SAL_EVENT_INFO_200")
-                return reply.data
+            status_dictionary = self.check_reply(reply)
+            if status_dictionary['accepted']:
+                logger.info("Position captured")
+                return int(reply.data), status_dictionary['code'], status_dictionary['message']
         except SerialException as e:
             logger.error(e)
-            logger.info("Command for device timed out - SAL_CMD_NOACK_-301")
-            raise
+            logger.info("Command for device timed out")
+            return 5, e
+
+    def enable(self):
+        self.port.open()
+        logger.info("port opened")
+
+    def disable(self):
+        self.port.close()
+        logger.info("port closed.")
+
+    def retrieve_status(self):
+        try:
+            reply = self.send("")
+            logger.debug(reply)
+            status_dictionary = self.check_reply(reply)
+            if status_dictionary['accepted']:
+                logger.info("Status captured")
+                return reply.device_status, status_dictionary['code'], status_dictionary['message']
+        except SerialException as e:
+            logger.error(e)
+            return 5, e
+
+    def stop(self):
+        try:
+            reply = self.send("stop")
+            logger.debug(reply)
+            status_dictionary = self.check_reply(reply)
+            if status_dictionary['accepted']:
+                logger.info("Device stopped")
+                return status_dictionary['code'], status_dictionary['message']
+        except SerialException as e:
+            logger.error(e)
+            return 5, e
+
+
+class SALCode:
+    def __init__(self, code_num):
+        self.sal_dictionary = {""}
 
 
 class LinearStageQueue:
```

Comment:
Is this a placeholder for something?

Reply:
It was, but I don't think its necessary anymore.